lib/exercism/client_version.rb

```diff
@@ -0,0 +1,21 @@
+module ExercismWeb
+  class ClientVersion
+    attr_accessor :user
+
+    def initialize(user:)
+      @user = user
+    end
+
+    def notice_when_client_outdated
+      unless dismissed?
+        "A new version of the exercism client is available (cli-2.4.0). Run <code>exercism upgrade</code> to get the latest hotness!"
```

Comment:
Oh, no you're right. The config option means we'll end up at 2.4.0. Yepp, for the experiment, let's keep it simple.